cai_causal_graph/time_series_causal_graph.py

```diff
@@ -118,6 +119,15 @@ def __init__(
         self._stationary_graph: Optional[TimeSeriesCausalGraph] = None
         self._minimal_graph: Optional[TimeSeriesCausalGraph] = None
 
+    # Overwrite type annotations for linting and code completion.
+    # from_adjacency_matrix: Callable[
+    #     [Arg(numpy.ndarray, 'adjacency'), DefaultArg(Optional[List[Union[NodeLike, int]]], 'node_names')],
+    #     TimeSeriesCausalGraph,
+    # ]
+    # from_skeleton: Callable[[Arg(Skeleton, 'skeleton')], TimeSeriesCausalGraph]
+    # from_networkx: Callable[[Arg(networkx.Graph, 'g')], TimeSeriesCausalGraph]
+    # from_gml_string: Callable[[Arg(str, 'gml')], TimeSeriesCausalGraph]
```

Comment:
@maxelliott-causalens I can't actually get this to work per the spec. Linting passes without it.